packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/pages/EditView/EditView.js

```diff
@@ -124,94 +115,85 @@ export function ReviewWorkflowsEditView() {
 
   useInjectReducer(REDUX_NAMESPACE, reducer);
 
-  useEffect(() => {
-    dispatch(setWorkflows({ status: workflowsData.status, data: workflowsData.data }));
-  }, [workflowsData.status, workflowsData.data, dispatch]);
-
-  useEffect(() => {
+  React.useEffect(() => {
     trackUsage('didViewWorkflow');
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  React.useEffect(() => {
+    dispatch(setAvailableContentTypes({ collectionTypes, singleTypes }));
+  }, [collectionTypes, singleTypes, dispatch]);
```

Comment:
If you think having these in the store, we could also pass them down directly to `WorkflowAttributes`, but we'd have a separate loading state.

Reply:
I suppose the app needs to be kept up to date i.e. the data will become stale if someone updates the content-types and another user navigates, but the hook could _always_ update redux...? 🤔

Or if we use `react-query` we can set a reasonable stale time?

Reply:
Or maybe this is fine

Reply:
So far we didn't have it in the store. I completely agree with you - could we make this a second step given the time we have for this? I guess we need to decide a layout for the future store anyway and I would be hesitant to access something from the CM namespace for a settings space, but then the question becomes where to store this.

Reply:
Yeah lets leave it for now, i think we can rethink all this when we look at data-fetching in the app holistically